src/main/java/org/opentripplanner/routing/algorithm/RaptorStateStore.java

```diff
@@ -1,31 +0,0 @@
-package org.opentripplanner.routing.algorithm;
-
-import gnu.trove.iterator.TObjectIntIterator;
-import gnu.trove.map.TObjectIntMap;
-
-import java.util.Collection;
-import java.util.Iterator;
-
-import org.opentripplanner.routing.core.State;
-import org.opentripplanner.routing.vertextype.TransitStop;
-
-/** Stores the states used in RAPTOR; allows easily switching to path-reconstructing store, a highly efficient optimal store, or McRAPTOR */
-public interface RaptorStateStore {
```

Comment:
Good to see this deleted. This is dead prototype code from R5 / Conveyal Analysis.